File: HLCObjKotlinAct4/src/Java/Act13/Act13Main.java
package Java.Act13;

import java.util.Scanner;

public class Act13Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String nombre,genero;
        int episodios;
        System.out.println("Escribe el nombre de anime");
        nombre = sc.nextLine();
        System.out.println("Escribe el numero de episodios");
        episodios = sc.nextInt();
        sc.nextLine();
        System.out.println("Escribe el genero");
        genero = sc.nextLine();

        Anime newAnime = new Anime(nombre,episodios,genero);
        System.out.println(newAnime);

    }
}

File: HLCObjKotlinAct4/src/Kotlin/Act13/Act13Main.kt
package Kotlin.Act13


fun main() {
    println("Escribe el nombre de anime")
    var nombre = readln()
        println("Escribe el numero de episodios")
    var episodios: Int = readln().toInt()
            println("Escribe el genero")
    var genero = readln()

        val newAnime = Anime(nombre, episodios, genero)
        println(newAnime)
    }

